src/test/java/net/kaczmarzyk/spring/data/jpa/utils/ThrowableAssertions.java

```diff
@@ -0,0 +1,19 @@
+package net.kaczmarzyk.spring.data.jpa.utils;
+
+import org.assertj.core.api.Assertions;
+
+public class ThrowableAssertions {
+	
+	public static <T extends Throwable> void assertThrows(Class<T> expectedType, ThrowableRun throwableRun, String expectedMessage) {
+		try {
+			throwableRun.run();
```

Comment:
I think you should add `fail("expected an exception to be thrown ...")` after this invocation, because right now it will pass if nothin is thrown.